README.md

```diff
@@ -19,10 +19,82 @@ git clone git@github.com:Jolg42/prisma-workshop.git
 Naviguez dans le répertoire du projet et installez les dépendances npm avec la commande suivante :
 
 ```
-cd prisma-workshop
+cd prisma-workshop-2021-fr
 npm install
 ```
 
 ## Atelier
 
-Vous pouvez trouver toutes les informations pour suivre l'atelier [ici](https://pris.ly/day2021-intro-fr).
+Vous pouvez trouver toutes les informations pour suivre l'atelier [ici](https://pris.ly/2021-intro-fr).
+
+# Une introduction pratique a Prisma (2021)
+
+# Bonjour!
+
+👋 Bienvenue à l'atelier **Une introduction pratique à Prisma**.
+
+🇬🇧 *English version here:*
+
+[A Practical Introduction to Prisma (2021)](https://github.com/nikolasburk/prisma-workshop)
+
+# Ressources
+
+**Raccourci pour cette page Notion** 
+
+[`https://pris.ly/2021-intro-fr`](https://pris.ly/2021-intro-fr)
+
+**GitHub starter** 
+
+[`https://github.com/prisma/prisma-workshop-2021-fr`](https://github.com/prisma/prisma-workshop-2021-fr)
+
+# Prérequis
+
+Afin d'accomplir avec succès les tâches de l'atelier, vous devez avoir :
+
+- [Node.js](https://nodejs.org/en/) installé sur votre machine (12.2.X / 14.X).
+- Il est recommandé (mais pas obligatoire) d'utiliser [VS Code](https://code.visualstudio.com/) pour les travaux pratiques.
+
+C'est tout 🙌 (*aucune connaissance préalable de SQL ou de Prisma n'est requise*).
+
+# Ce que vous allez faire
+
+Dans cet atelier, vous apprendrez les différents flux de travail qu'il est utile de connaître quand on utilise Prisma.
+
+Vous commencerez par configurer Prisma avec une base de données SQLite, vous apprendrez à modéliser des données avec Prisma et à effectuer des migrations de bases de données (*leçon 1*).
+
+Ensuite, vous découvrirez Prisma Client, un *query builder* *type-safe* qui peut être utilisé pour interroger votre base de données. Vous allez explorer différentes requêtes, du simple CRUD aux requêtes de relation, en passant par les filtres et la pagination (*leçon 2*).
+
+Ensuite, vous apprendrez comment utiliser Prisma Client pour implémenter les routes d'une API REST (*leçon 3*).
+
+Enfin, nous verrons comment utiliser Prisma Client pour implémenter les résolveurs d'une API GraphQL (*leçon 4*).
+
+# Agenda
```

Comment:
We can remove the Agenda section as it's no longer needed

Reply:
I have updated this